src/panels/logbook/ha-logbook.ts

```diff
@@ -254,6 +265,126 @@ class HaLogbook extends LitElement {
     this._savedScrollPos = (e.target as HTMLDivElement).scrollTop;
   }
 
+  private _formatEventBy(item: LogbookEntry, seenEntities: string[]) {
+    if (item.context_event_type === "call_service") {
+      return `${this.hass.localize("ui.components.logbook.from_service")} ${
+        item.context_domain
+      }.${item.context_service}`;
+    }
+    if (
+      item.context_event_type === "automation_triggered" &&
+      item.context_entity_id
```

Comment:
If event type is `automation_triggered`, it currently still processes rest of this method if there is no `item.context_entity_id`. Does that make sense?

Reply:
It shouldn't be possible to not have `context_entity_id` for an automation since automations always have an entity_id in the row

Reply:
Ok, then let's skip this check and instead use `!`

Reply:
5b9e1d4cf3c1ae2ede13f2f4a11c1c7b7703120a